test/EmptyApp.FunctionalTests/Startup.cs

```diff
@@ -25,7 +25,7 @@ public void ConfigureServices(IServiceCollection services)
 
             var builder = new ConfigurationBuilder();
             builder.AddApplicationInsightsSettings(instrumentationKey: "Foo");
-            services.AddApplicationInsightsTelemetry(builder.Build());
+            services.AddApplicationInsightsTelemetry(builder.Build(), false);
```

Comment:
Can we leave this to true, and create a new functional test that would test if telemetry client is sending out the telemetry right? The telemetry client that we use today in the test is the one that we  get from services. At the least, we can send duplicate telemetry from two telemetry clients, and verify if we get it right.